### PR TITLE
new plugin WebScreenShareFixes: remove low stream bitrate limit

### PR DESCRIPTION
**Pull request by kaitlynkittyy**
you can read i hope, the title explains. also make sure to merge this before [Vesktop #563]( https://github.com/Vencord/Vesktop/pull/563)

### CODE REVIEW

**Comment by kaitlynkittyy** [comment]
@Vendicated :pleading_face: did i do it right?

File: src/plugins/WebScreenShareFixes.web/index.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    name: "WebScreenShareFixes",
+    authors: [Devs.Kaitlyn],
+    description: "Removes 2500kbps bitrate cap on chromium and vesktop clients.",
+    enabledByDefault: true,
+    patches: [
+        {
+            find: "x-google-max-bitrate",
+            replacement: [
+                {
+                    // eslint-disable-next-line no-useless-escape
```
Comment (Vendicated) [inline src/plugins/WebScreenShareFixes.web/index.ts:20]:
```suggestion
```

**Comment by Vendicated** [comment]
please fix the filename casing. it should be camelCase!

File: src/plugins/WebScreenShareFixes.web/index.ts (hunk 1 above)
Comment (kaitlynkittyy) [reply src/plugins/WebScreenShareFixes.web/index.ts]: oop, my bad
its a leftover from vesktop, as it was needed in that source

**Comment by kaitlynkittyy** [comment]
> please fix the filename casing. it should be camelCase!

tell that to wikisearch 
image

File: src/plugins/webScreenShareFixes.web/index.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        {
+            find: "x-google-max-bitrate",
+            replacement: [
+                {
+                    match: /"x-google-max-bitrate=".concat\(\i\)/,
+                    replace: '"x-google-max-bitrate=".concat("80_000")'
+                },
+                {
+                    match: /;level-asymmetry-allowed=1/,
+                    replace: ";b=AS:800000;level-asymmetry-allowed=1"
```
Comment (Vendicated) [inline src/plugins/webScreenShareFixes.web/index.ts:25]: could you test if this breaks stuff when applied twice?

Comment (kaitlynkittyy) [reply src/plugins/webScreenShareFixes.web/index.ts:25]: will do

Comment (kaitlynkittyy) [reply src/plugins/webScreenShareFixes.web/index.ts:25]: I'm not having any issues, the only thing I see is this: 
image

Comment (kaitlynkittyy) [reply src/plugins/webScreenShareFixes.web/index.ts:25]: And yes, I'm not locked to 2500

**Review by Vendicated — APPROVED** [review]

**Comment by kaitlynkittyy** [comment]
btw this should be fully ready to merge

**Comment by kaitlynkittyy** [comment]
@Vendicated mlem

**Comment by GingerBobRoss** [comment]
I wanted to report that I have this plugin installed on Vesktop in Cachy-OS, but it does not appear to be working.
When watching friends stream, the max bandwidth is still 2500kbps

**Pull request merged**